tests/helpers/participants.ts

```diff
@@ -164,25 +164,6 @@ export async function toggleMuteAndCheck(testee: Participant, observer: Particip
     await testee.getFilmstrip().assertAudioMuteIconIsDisplayed(testee);
 }
 
-/**
- * Parse a JID string.
- * @param str the string to parse.
- */
-export function parseJid(str: string): {
```

Comment:
I think you did well not introducing the extra dependency, sorry for the noise. I'd undo this.